client/log_client_test.go

```diff
@@ -54,12 +55,18 @@ func TestGetByIndex(t *testing.T) {
 		t.Fatal(err)
 	}
 	defer env.Close()
-	logID, err := env.CreateLog()
+
+	tree, err := CreateAndInitTree(ctx, &trillian.CreateTreeRequest{
+		Tree: stestonly.LogTree,
+	}, env.Admin, nil, env.Log)
```

Comment:
I'm finding this a bit hard to read because of the  `}` being outdented as far as this.  Presumably gofmt has done that.
Would it be nicer to read like this:
```
tree, err := CreateAndInitTree(ctx, &trillian.CreateTreeRequest{Tree: stestonly.LogTree},
        env.Admin, nil, env.Log)
```

Reply:
Or like this:
```
tree, err := CreateAndInitTree(ctx,
        &trillian.CreateTreeRequest{Tree: stestonly.LogTree},
        env.Admin, nil, env.Log)
```

Reply:
I'd only split and indent it the way you did if there were several fields being declared in `CreateTreeRequest`

Reply:
Thanks, the second version seems much clearer.